spec/aws-record/record/item_collection_spec.rb

```diff
@@ -77,6 +77,23 @@ module Record
         end
       end
 
+      describe "#new_record" do
+        it 'marks records fetched from a client call as not being new' do
```

Comment:
We should go ahead and add the "obvious" test that something created by hand is marked as a new record. Part of the test coverage notion is to avoid surprise regressions later.